src/viewer/Viewer.js

```diff
@@ -1,11 +1,34 @@
 import React, { Component } from 'react'
 import { Nav, Navbar, Container } from "react-bootstrap"
-import { setLoggedIn } from '../App'
+import { setLoggedIn } from '../Login'
 import FileUpload from './upload/FileUpload'
+import CornerstoneViewer from './CornerstoneViewer'
 import "./Viewer.css"
 
 class Viewer extends Component {
```

Comment:
Can you update this container to a functional component instead of a class. This is to stay consistent with all the components.